conductr_cli/test/test_conduct_events.py

```diff
@@ -33,32 +33,46 @@ def test_no_events(self):
     def test_multiple_events(self):
         http_method = self.respond_with(text="""[
             {
+                "@appname": "ConductR",
                 "@cee": {
+                    "body": {
+                        "bundleName": "visualizer",
+                        "message": "Load bundle requested: requestId=cba938cd-860e-41a4-9cbe-2c677feaca20, bundleName=visualizer",
+                        "requestId": "cba938cd-860e-41a4-9cbe-2c677feaca20"
+                    },
                     "head": {
                         "contentType": "conductr.loadScheduler.loadBundleRequested"
                     },
-                    "body": {
-                        "time": "Today 12:54:30",
-                        "requestId": "req123",
-                        "bundleName": "bundle-name",
-                        "message": "Load bundle requested: requestId=req123, bundleName=bundle-name"
-                    },
                     "tag": "conductr.loadScheduler.loadBundleRequested"
-                }
+                },
+                "@facility": "user",
+                "@hostname": "ubuntu",
+                "@msgid": "-",
+                "@procid": "-",
+                "@severity": "info",
+                "@timestamp": "2015-07-29T12:10:34+10:00",
+                "@sd": "[mdc source=\\\"conductr-event(akka://conductr)\\\" class=\\\"class akka.event.DummyClassForStringSources\\\"]"
             },
             {
+                "@appname": "ConductR",
                 "@cee": {
+                    "body": {
+                        "bundleName": "visualizer",
+                        "message": "Bundle written: requestId=cba938cd-860e-41a4-9cbe-2c677feaca20, bundleName=visualizer",
+                        "requestId": "cba938cd-860e-41a4-9cbe-2c677feaca20"
+                    },
                     "head": {
                         "contentType": "conductr.loadExecutor.bundleWritten"
                     },
-                    "body": {
-                        "time": "Today 12:54:36",
-                        "requestId": "req123",
-                        "bundleName": "bundle-name",
-                        "message": "Bundle written: requestId=req123, bundleName=bundle-name"
-                    },
                     "tag": "conductr.loadExecutor.bundleWritten"
-                }
+                },
+                "@facility": "user",
+                "@hostname": "ubuntu",
+                "@msgid": "-",
+                "@procid": "-",
+                "@severity": "info",
+                "@timestamp": "2015-07-29T12:10:34+15:00",
+                "@sd": "[mdc source=\\\"conductr-event(akka://conductr)\\\" class=\\\"class akka.event.DummyClassForStringSources\\\"]"
```

Comment:
Change the json we use in the test to reflect an actual example that might be sent from the logging endpoint